controllers/module_reconciler.go

```diff
@@ -57,7 +57,7 @@ type ModuleReconciler struct {
 
 func NewModuleReconciler(
 	client client.Client,
-	buildAPI build.Manager,
+	buildAPI []build.Manager,
```

Comment:
As discussed in another comment, I would have a separate interface for the signing feature.